datafusion/sql/src/expr/mod.rs

```diff
@@ -1195,4 +1195,25 @@ mod tests {
     test_stack_overflow!(2048);
     test_stack_overflow!(4096);
     test_stack_overflow!(8192);
+    #[test]
+    fn test_sql_to_expr_with_alias() {
+        let schema = DFSchema::empty();
+        let mut planner_context = PlannerContext::default();
+
+        let expr_str = "int_col as int_col_alias";
```

Comment:
thanks @Eason0729, IMHO this looks good, can we get another test which will test expression with function as well, something like `SUM(int_col) as sum_int_col` ?